__tests__/exportPlatform.test.js

```diff
@@ -234,156 +234,210 @@ describe('exportPlatform', () => {
     expect(actual).to.eql(expected);
   });
 
-  // describe('token references without .value', async () => {
```

Comment:
These tests were disabled for some reason, so I re-enabled them, since they do work fine and provide coverage